src/Commands/CheckTranslationsCommand.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace NextApps\PoeditorSync\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Collection;
+use NextApps\PoeditorSync\Poeditor\Poeditor;
+use NextApps\PoeditorSync\Translations\TranslationManager;
+
+class CheckTranslationsCommand extends Command
+{
+    protected $signature = 'poeditor:check';
+
+    protected $description = 'Check if local translations match the ones on POEditor';
+
+    public function handle() : int
+    {
+        $this->getLocales()->each(function ($locale, $key) {
+            $translations = app(Poeditor::class)->download(is_string($key) ? $key : $locale);
+
+            collect($locale)->each(function ($internalLocale) use ($translations) {
+                $this->info("Checking translations for {$internalLocale}");
+                $valid = app(TranslationManager::class)->checkTranslations(collect($translations), $internalLocale);
+
+                if (! $valid) {
+                    $this->error("The translations for {$internalLocale} do not match the ones on POEditor");
+                }
+            });
+        });
```

Comment:
```suggestion
        $isOutdated = $this->getLocales()->map(function ($locale, $key) {

            $poeditorTranslations = app(Poeditor::class)->download(is_string($key) ? $key : $locale);

            return collect($locale)->map(function ($internalLocale) use ($poeditorTranslations) {
                $localTranslations = app(TranslationManager::class)->getTranslations($internalLocale);

                if ($poeditorTranslations === $localTranslations) {
                    return true;
                }
                
                $this->error("The translations for {$internalLocale} do not match the ones on POEditor.");
                
                return false;
            });
        })->flatten()->contains(false);
        
        if ($isOutdated) {
        	return Command::FAILED;
        }
        
        $this->info('All translations match the ones on POEditor!')
        
        return Command::SUCCESS;
```
if a dedicated compare method is necessary, I would add that to this command. TranslationManager should only be for getting or setting translations